internal/osutils/autostart/autostart_linux.go

```diff
@@ -96,6 +81,23 @@ func autostartPath(name string, opts Options) (string, error) {
 	return "", nil
 }
 
+func upgrade(exec string, opts Options) error {
```

Comment:
It doesn't look like you're using `exec`?

Reply:
Yeah, that's too defensive of a definition decision, on my part, for an internal package. The param is removed.